### PR TITLE
travis: use shorter channel notation

### PR DESCRIPTION
**Pull request by zimbatm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 language: nix
 env:
-  - NIXPKGS_CHANNEL=17.09
-  - NIXPKGS_CHANNEL=unstable
+  - NIXPKGS_CHANNEL=nixos-17.09
```
Comment (moretea) [inline .travis.yml]: Maybe also bump the stable version to 18.09?

**Review by moretea — APPROVED** [review]
LGTM!

File: .travis.yml (hunk 1 above)
Comment (zimbatm) [reply .travis.yml]: maybe, I like that yarn2nix is back-compat with at least 2 versions

Comment (zimbatm) [reply .travis.yml]: did a compromise in 
62b3418

**Pull request merged**